src/adapter/src/coord/sequencer.rs

```diff
@@ -2562,12 +2562,7 @@ impl<S: Append + 'static> Coordinator<S> {
                 )?;
                 let since = self.least_valid_read(&id_bundle).elements().to_vec();
                 let upper = self.least_valid_write(&id_bundle).elements().to_vec();
-                let has_table = id_bundle.iter().any(|id| self.catalog.uses_tables(id));
-                let table_read_ts = if has_table {
-                    Some(self.get_local_read_ts())
-                } else {
-                    None
-                };
+                let respond_immediately = upper.iter().all(|upper| upper > &timestamp);
```

Comment:
@frankmcsherry or @mjibson I'm actually a bit confused by this now that I think of it. Should it be `all` or `any` or "we know that there will only ever be a single value here so we don't care".

Reply:
If `upper` is an `Antichain`, it is idiomatic to do `!upper.less_equal(&timestamp)`.

Reply:
Switched